public/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="shortcut icon" href="">
```

Comment:
Not sure what the plan is for this?

Reply:
I was getting a weird 404 in my console.  I googled and it wasn't any real problem, but the suggested solution had this so it wouldn't throw and error anymore.

Reply:
Interesting. Please let me know if you find out more about what was happening